backends/jms/pom.xml

```diff
@@ -41,6 +41,12 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+
+        <!-- Avoid warnings during build -->
+        <dependency>
```

Comment:
It's provided, as it is defined as provided in the dependency management of the aggregator